schematic-compiler/src/main/java/net/simonvt/schematic/compiler/TableWriter.java

```diff
@@ -282,45 +282,30 @@ public void createValuesBuilder(Filer filer, String outPackage) throws IOExcepti
 
       switch (type) {
         case INTEGER:
-          MethodSpec intSpec = MethodSpec.methodBuilder(elmName)
-              .addModifiers(Modifier.PUBLIC)
-              .returns(ClassName.get(valuesPackage, className))
-              .addParameter(int.class, "value")
-              .addStatement("values.put($T.$L, value)", ClassName.get(columnsClass), column)
-              .addStatement("return this")
-              .build();
-          valuesBuilder.addMethod(intSpec);
-
-          MethodSpec longSpec = MethodSpec.methodBuilder(elmName)
-              .addModifiers(Modifier.PUBLIC)
-              .returns(ClassName.get(valuesPackage, className))
-              .addParameter(long.class, "value")
-              .addStatement("values.put($T.$L, value)", ClassName.get(columnsClass), column)
-              .addStatement("return this")
-              .build();
-          valuesBuilder.addMethod(longSpec);
+          valuesBuilder.addMethod(
+              makePutMethodSpec(valuesPackage, className, elmName, column, int.class));
+          valuesBuilder.addMethod(
+              makePutMethodSpec(valuesPackage, className, elmName, column, long.class));
```

Comment:
int and long are redundant, there are only Integer and Long methods on ContentValues.